x-pack/plugins/alerting/server/task_runner/types.ts

```diff
@@ -96,7 +97,7 @@ export type Executable<
   ActionGroupIds extends string,
   RecoveryActionGroupId extends string
 > = {
-  action: RuleAction;
+  action: RuleAction | RuleSystemAction;
```

Comment:
Now that we removed the `type`, should we remove the `RuleSystemAction` from here?